frontend/src/views/CreateAccountPageView.tsx

```diff
@@ -31,8 +32,8 @@ const CreateAccountPage: React.FC = () => {
 
 		// create account on firebase
 		try {
-			await createUserWithEmailAndPassword(auth, email, password);
-
+			const user = await createUserWithEmailAndPassword(auth, email, password);
+			await addToUserCollection(user);
```

Comment:
not exactly relevant to this PR, could be grouped together with a future PR with the
1. 'username' field at the create account page; and
2. 'welcome, <username>' message at the questions page;